base/docs/helpdb.jl

```diff
@@ -2024,13 +2024,6 @@ Reseed the random number generator. If a `seed` is provided, the RNG will give a
 srand
 
 doc"""
-    isexecutable(path) -> Bool
-
-Returns `true` if the current user has permission to execute `path`, `false` otherwise.
```

Comment:
I think it's most clearly documented at https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#improving-documentation right now, but yes kind of confusing at the moment.

+1 to merge when green.